#185 【美食作家王刚】去华农家摸鱼烤竹鼠 (#195)
immoonancient/YTSubtitles: subtitles/wang-gang/20190915-catch-bbq.txt

#https://youtu.be/S4cHtPsK0VM

0:00:00.900,0:00:07.120
#现在我们一大帮人去他（华农）兄弟家抓鱼

> Now the whole group of us are going to catch the fish at our buddies' (Huanong Brothers) place.

Reviewer: ```suggestion
Now the whole group of us are going to catch fish at our buddies' (Huanong Brothers) place.
```

Reviewer: 个人感觉没有the好一些？

0:00:07.660,0:00:12.980
#有四伯、苏良兄还有他的一些兄弟

> Here are my uncle, Suliang and some of his buddies we well.

Reviewer: ```suggestion
Here are my uncle, Suliang (Huanong) and some of his buddies we well.
```

0:00:13.540,0:00:18.380
#等下抓回来的时候让这位大厨弄点给我们吃一下
After catching the fish, we are going to have this Chef to serve us a feast.

0:00:18.640,0:00:19.920
#那必须的
Sure! My pleasure.

0:00:20.560,0:00:22.400
#但是我弄的都很辣哦
But I am going to make it very spicy.

0:00:22.400,0:00:25.740
#没事我挺能吃辣的
No worries. I can eat rather spicy food.

0:00:25.820,0:00:31.040
#现在慢慢都吃辣椒了以前都不吃的
I didn't eat too much chili before, but now I am getting used to it.

0:00:32.120,0:00:36.300
#我兄弟带我来他们家自己的鱼塘里面
My buddy is taking me to their own pond.

0:00:37.560,0:00:39.340
#把水放干了好抓鱼
We'll be draining away the water first so that it becomes easier to catch the fish.

0:00:39.880,0:00:43.500
#他说人太多了不好（意思）去祸害别的兄弟
He said that he is not going to loot other buddies when so many people are around.

0:00:43.600,0:00:49.420
#那就只能现在在他们鱼塘里面把水放干了抓点鱼
So he has no choice but draining his own pond so that we can catch some fish.

0:00:53.080,0:00:54.420
#你们家狗也来了
Your dogs are coming too.

0:01:04.280,0:01:06.460
#你这里都放了些什么鱼呢

> What fish have you put here?

Reviewer: ```suggestion
What fish have you put in here?
```

0:01:06.500,0:01:07.540
#蛇头鱼
Snakehead fish

0:01:08.520,0:01:10.160
#就是那个黑鱼（生鱼）
I mean the black fish (sheng yu).

0:01:11.740,0:01:13.380
#有些叫生鱼有些叫黑鱼
It's sometimes called "sheng yu" or "black fish" at other time.

0:01:14.240,0:01:15.680
#估计都只剩下黑鱼了
I think only the black fish remain.

0:01:16.240,0:01:20.340
#我放了好多小鱼下去肯定被蛇头鱼吃掉了
I have put a lot of small fish but I am sure that the snakehead fish have eaten them all,

0:01:22.060,0:01:23.060
#因为（黑鱼）吃肉的
as the back fish are carnivores.

0:01:24.540,0:01:26.520
#黄鳝泥鳅都放了好多下去
I have put a lot of eel and loach.

0:01:26.540,0:01:29.740
#我以为你们这边的鱼塘都是很大面积的

> I thought that the ponds here were the very large ones,

Reviewer: ```suggestion
I thought that the ponds here were very large,
```

It's here. Got it!

0:04:11.360,0:04:12.160
#两条
We got two!

0:04:12.860,0:04:14.000
#哇塞好多鱼啊
Wa! We have a lot of fish!

0:04:17.680,0:04:19.060
#这前方有一条
There's one in front of us.

0:04:26.120,0:04:28.080
#这鱼有
We now should have...

0:04:29.360,0:04:30.440
#有五六斤
5 to 6 jin (1jin = 500 grams) of fish.

0:04:31.340,0:04:32.380
#有五六斤啊
Is it that much?

0:04:32.380,0:04:33.540
#真的有五六斤
Yeah! Really

0:04:35.640,0:04:37.700
#这个我们叫生鱼 我们叫蛇头鱼
We call this fish "sheng yu".
And we call it snakehead fish.

0:04:37.700,0:04:38.540
#好瘦啊
It's very skinny.

0:04:38.540,0:04:39.700
#好吃这种鱼
This fish is tasty.

0:04:40.180,0:04:41.140
#好瘦啊这个鱼
This fish is very skinny.

0:04:42.040,0:04:42.960
#你脚那里还有一条
There's one more fish next to your foot.

0:04:43.380,0:04:44.900
#能活不能活呐
Can it survive?

0:04:45.480,0:04:49.500
#哟哟哟又把它抓回来了
Yo yo yo! I catch it back again.

0:04:51.960,0:04:53.760
#可以活的把它放回去
Yes, it can. Let's release it.

0:04:54.280,0:04:56.540
#这种小鱼不好吃的
This kind of small fish....tastes nothing!

0:04:58.020,0:05:00.740
#走啦走啦把鱼抱回去
Let's go, let's go. I'll take it back.

0:05:14.160,0:05:15.920
#兄弟带我弄了半天鱼
My buddy took me to catch the fish for a good whole day.

0:05:16.940,0:05:18.220
#都快闪到腰了

> My back has almost twisted.

Reviewer: ```suggestion
My waist is almost hurt.
```

0:06:18.580,0:06:19.760
#可以板栗吃得饱
No! You'll be stuffed with chestnuts,

0:06:20.180,0:06:22.160
#吃不饱再带去兄弟家吃饭
or you can come to my buddy to have a feast.

0:06:26.260,0:06:29.820
#等一下要去你们圈里面抓2只竹鼠来
You are going to catch two bamboo rats from your shred, aren't you?

0:06:29.900,0:06:30.740
#可以 没问题
Sure! No problem.

0:06:30.800,0:06:31.520
#嗯
Good.

0:06:31.800,0:06:34.620
#其实竹鼠我已经吃过了我想吃那个猪
In fact, I have already tried bamboo rats. What I want is your pig.

0:06:34.620,0:06:38.660
#猪就等它长大一点现在太小了
They're too small now. Let's wait until they become bigger.

0:06:39.080,0:06:41.100
#肉很嫩不好吃要老一点才香
It doesn't taste good when the pork is too soft. More mature and chewy pork,

0:06:41.900,0:06:43.380
#才有嚼劲
is more tasty.

0:06:43.380,0:06:44.040
#好吧
Alright, trust you.

0:06:47.360,0:06:48.220
#哎哟
Ayo!

0:06:48.580,0:06:49.760
#屁股那么大吗
Is my butt that huge?

0:06:50.520,0:06:51.480
#别搞坏了
Don't break it.

0:06:51.480,0:06:53.020
#管他呢不是我家的
Never mind. It's not mine anyway.

0:06:53.020,0:06:55.460
#嘿嘿又不是你家的
He he. Did you say it's not yours?

0:06:56.700,0:06:58.820
#前进
March on!

0:06:59.120,0:07:01.740
#妹妹你坐船头哦
Singing: "Cutie, you sit at the front"

0:07:01.740,0:07:03.680
#哥哥的岸上走

> Singing: "And bro, you walk on the shore"

Reviewer: ```suggestion
Singing: "And bro, you walk on the riverbank"
```

0:07:04.120,0:07:05.640
#我应该去岸上走吧

> Should I walk on the shore?

Reviewer: ```suggestion
Should I walk on the riverbank?
```

0:07:06.100,0:07:06.920
#嘿嘿
He he

0:07:35.240,0:07:36.480
#小哥上鱼了
Hey man, you got a fish!

0:07:36.500,0:07:37.960
#看这条鱼
Have a look at this fish.

0:07:39.040,0:07:40.180
#这个可能有
I think it weighs...

0:07:40.820,0:07:42.900
#1斤多不到2斤

> at least a jin, but not 2.

Reviewer: ```suggestion
at least one jin, but less than 2.
```

0:08:01.000,0:08:01.500
#来
Take one.

0:08:01.500,0:08:02.400
#里面还有吗
Are there still any?

0:08:03.260,0:08:04.260
#直接这样吃呀
Is it eaten right away?

0:08:04.600,0:08:05.180
#不是
Nope!

0:08:05.460,0:08:07.420
#吃里面的肉要把它剥开
You need to peal it. The edible part is inside.

0:08:09.740,0:08:11.240
#这个肉就那么一丢丢
The edible part is really tiny.

0:08:11.240,0:08:12.560
#这个不是很熟的
It's still not ready.

0:08:12.820,0:08:14.440
#也被我拧下来啦
But I have still picked it.

0:08:16.380,0:08:19.620
#嗯 真的无色无味
Aha! It's really colourless and tasteless.

0:08:20.320,0:08:21.960
#就是那种甘甘甜甜的
It's like, savory sweet.

0:08:21.960,0:08:22.880
#嗯
Yea

0:08:28.360,0:08:29.940
#太阳差不多要下山了 走吧
Let's go! The sun is already going down.

0:08:29.940,0:08:30.440
#嗯
Yep

0:08:30.740,0:08:32.040
#看一下你的竹鼠
Let's have a look at your bamboo roots.

0:08:32.040,0:08:33.100
#烤竹鼠
Grill them.

0:08:33.160,0:08:33.800
#走吧
Let's go!

0:08:37.120,0:08:41.020
#我这兄弟终于愿意弄竹鼠来给我吃了
Finally, my buddy is willing to let me have his bamboo rats.

0:08:42.640,0:08:44.140
#这3只竹鼠
These three bamboo rats...

0:08:44.580,0:08:46.360
#你看这个就是很不听话的
Look! This is very naughty.

0:08:46.360,0:08:47.640
#老是动来动去

> It's always moving around.

Reviewer: ```suggestion
It's restless.
```

0:08:48.080,0:08:49.900
#都是公竹鼠吗
They are all male, aren't they?

0:08:49.900,0:08:50.700
#全是公的
Yea, all are males.

0:08:50.700,0:08:52.980
#母的我不会杀的
The females are not for food

0:08:53.320,0:08:54.340
#要留着下崽
but for giving birth to new rats.

0:08:54.940,0:08:55.440
#好吧
Got it.

0:08:55.440,0:08:59.120
#上次你说你那里面都是母的没有公的
You said last time that you have only females and no more males.

0:08:59.120,0:09:00.660
#我现在的竹鼠啊
My ratio now,

0:09:00.660,0:09:03.160
#1个公的配7个母的
is one male to seven females.

0:09:03.520,0:09:05.100
#比例呀已经比成这样子啦
What a ratio!

0:09:05.100,0:09:08.400
#不能再向他们动手啦
I'll need to spare the males from now on.

0:09:08.400,0:09:11.280
#到时候就是1个公的配10个母的
Otherwise, each male will need to mate with 10 females.

0:09:12.700,0:09:13.760
#嗯 可以
Um, alright.

0:09:13.960,0:09:14.800
#开动
Let's get started.

0:09:30.600,0:09:31.640
#好
Alright!

0:09:39.540,0:09:40.740
#车上有音乐噢
There's music from the car.

0:09:40.740,0:09:42.520
#DJ搞起来
Here comes the DJ!

0:09:51.205,0:09:52.255
#It's ok
It's ok.

0:09:52.740,0:09:53.820
#看我们这个
See what we have!

0:09:54.320,0:09:58.600
#我已经研究透了这个竹鼠要躲几刀才烤得够快

> After my thorough research, you need to cut the bamboo rats several times in order to shorten the grilling time.

Reviewer: ```suggestion
I have figured out how many times you need to chop the bamboo rats to shorten the grilling time.
```
